tests/snapshot_tests/test_snapshots.py

```diff
@@ -1448,3 +1448,11 @@ def test_command_palette_key_change(snap_compare):
 def test_split(snap_compare):
     """Test split rule."""
     assert snap_compare(SNAPSHOT_APPS_DIR / "split.py", terminal_size=(100, 30))
+
+
+def test_scroll_page_down(snap_compare):
+    """Regression test for https://github.com/Textualize/textual/issues/4914"""
+    # Should show 25 at the top
```

Comment:
Just a reminder that if you write the comment in the docstring, it'll  appear in the snapshot report now which is really handy if the test fails.

Also, you can inline the app if you want and just pass in the instance rather than putting it in a separate file.